modules/widgets/top-posts.php

```diff
@@ -251,6 +251,8 @@ function widget( $args, $instance ) {
 				/** This filter is documented in modules/stats.php */
 				'gravatar_default' => apply_filters( 'jetpack_static_url', set_url_scheme( 'https://en.wordpress.com/i/logo/white-gray-80.png' ) ),
 				'avatar_size' => 40,
+				'width' => null,
+				'height' => null,
```

Comment:
Hey @singerb, I tried to make the code bulletproof, but I totally forgot about the filter docs ( shame on me :smile: ). Thanks for your comment, the new options have been added.